.github/workflows/agw-workflow.yml

```diff
@@ -95,6 +105,9 @@ jobs:
           SLACK_FOOTER: ' '
 
   session_manager_test:
+    needs: path_filter
+    if: ${{ needs.path_filter.outputs.should_skip != 'true' }}
```

Comment:
replace with `== 'false'`

Reply:
@quentinDERORY - Addressed the comments and pushed the changes